app/views/languages/_form.html.erb

```diff
@@ -2,51 +2,50 @@
 
 <%= form_for(@language, html: { class: "post" }) do |f| %>
 
-  <p class="required notice">* <%= t('.required_notice', default: "Required information") %></p>
+  <p class="required notice">* <%= t(".required_notice") %></p>
 
   <fieldset>
     <dl>
       <dt class="required">
-        <%= f.label :name, t('.name', default: "Name") + " *" %>
+        <%= f.label :name, "#{t('.fieldset.name')} *" %>
```

Comment:
```suggestion
        <%= f.label :name, "#{t(".fieldset.name")} *" %>
```

I would also remove the `.fieldset` prefix from the fieldset locale keys because they're already prefixed with `form`. But thank you for the i18n work!

Reply:
it seems like reviewdog wants the quotes to be single quotes

Reply:
I see, thank you!